lession13_dom_event/exercise-4-displayMenu/main.js

```diff
@@ -0,0 +1,218 @@
+                      children: [
+                          { name: 'About 1.1.1' }
+                      ]
+                  },
+                  {
+                      name: 'About 1.2',
+                      children: [
+                          {
+                              name: 'About 1.2.1',
+                              children: [
+                                  { name: 'About 1.2.1.1' },
+                                  { name: 'About 1.2.1.2' },
+                                  { name: 'About 1.2.1.3' }
+                              ]
+                          }
+                      ]
+                  }
+              ]
+          },
+          {
+              name: 'Home 2',
+              children: [
+                  {
+                      name: 'Home 2.1'
+                  }
+              ]
+          }
+      ]
+  }
+]
+const [
+  {
+      name: home,
+      children: [
+          {
+              name: home1,
+              children: [
+                  {
+                      name: home1_1,
+                      children: [
+                          { name: home1_1_1 }
+                      ]
+                  },
+                  {
+                      name: home1_2,
+                      children: [
+                          { name: home1_2_1 }
+                      ]
+                  }
+              ]
+          },
+          {
+              name: home2,
+              children: [
+                  {
+                      name: home2_1
+                  }
+              ]
+          },
+          {
+              name: home3,
+              children: [
+                  {
+                      name: home3_1
+                  }
+              ]
+          }
+      ]
+  },
+  {
+      name: about,
+      children: [
+          {
+              name: about1,
+              children: [
+                  {
+                      name: about1_1,
+                      children: [
+                          { name: about1_1_1 }
+                      ]
+                  },
+                  {
+                      name: about1_2,
+                      children: [
+                          {
+                              name: about1_2_1,
+                              children: [
+                                  { name: about1_2_1_1 },
+                                  { name: about1_2_1_2 },
+                                  { name: about1_2_1_3 }
+                              ]
+                          }
+                      ]
+                  }
+              ]
+          },
+          {
+              name: about_home2,
+              children: [
+                  {
+                      name: about_home2_1
+                  }
+              ]
+          }
+      ]
+  }
+] = menu
+const displayMenu = document.getElementById('menu')
+const showMenu = menu => {
+  displayMenu.innerHTML = `
+  <ul>
+  <li id ="home">  ${home}
```

Comment:
Làm cái kiểu gì đây?
array trong đề chỉ là 1 ví dụ,
```suggestion
  <li id="home">  ${home}
```